slack-sup/models/round.rb

```diff
@@ -22,7 +22,7 @@ def to_s
 
   def ask?(dt = 3.days)
     return false if asked_at
-    ran_at && ran_at + dt <= Time.now.utc
+    ran_at && ran_at <= dt.ago
```

Comment:
This now seems incorrectly hard-coded at 3 days. If I want the followup day to be tomorrow after the sup, this will prevent it from happening for 2 more days, no?

Reply:
so the caller [here](https://github.com/ashkan18/slack-sup/blob/configurable-followup-day/slack-sup/models/team.rb#L85) now passes teams follow up day, so should be fine right?

Reply:
Yes, but remove the default altogether because where before the logic was "x days later", the logic now is on a certain day of the week. I would probably remove completely the `dt` argument here.